locale/fr_CA/admin.xml

```diff
@@ -97,4 +97,9 @@
 	<message key="admin.fileLoader.fileProcessed">Le fichier {$filename} a été traité et archivé.</message>
 	<message key="admin.fileLoader">Chargement du fichier</message>
 	<message key="admin.scheduledTask.openAccessNotification">Notification d'accès libre</message>
+	<message key="admin.scheduledTask.downloadLog">Votre installation de OJS a terminé l'exécution de cette tâche. Vous pouvez télécharger le journal d'exécution ici: {$url}</message>
+	<message key="admin.scheduledTask.confirmClearLogs">Êtes-vous certain de vouloir supprimer le journal d'exécution des tâches planifiées ?</message>
```

Comment:
The blank at the end ("planifiées ?") - is that on purpose? Haven't seen it elsewhere in this locale.